src/components/calendar/bl-calendar.ts

```diff
@@ -521,7 +521,9 @@ export default class BlCalendar extends LitElement {
             variant="tertiary"
```

Comment:
label="Previous Calendar View" for example

Reply:
Oh! I missed the label property of button elements, Thank you. I'm on it

Reply:
I’d suggest updating it to “next/previous year” if possible, as the buttons specifically navigate between **years**. The current version works as well, though.

Reply:
I thought about changing the button label to next/previous year, but can also switch between months with this button. So I thought the most logical is the previous/next calendar view.

Reply:
I think we can make it dynamic in this case depending on the use and state.

Reply:
Yes, this make sense. I'm on it.